### ISSUE TITLE
Compilation error on G++6 (flexible array member)

### ISSUE THREAD

**Issue by ignatenkobrain** [Nitrokey/libnitrokey#49]
```c
In file included from /home/brain/rpmbuild/SPECS/libnitrokey/include/NitrokeyManager.h:6:0,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/NitrokeyManager.cc:3:
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:137:27: error: flexible array member ‘nitrokey::proto::EmptyPayload::_data’ in an otherwise empty ‘struct nitrokey::proto::EmptyPayload’
             uint8_t _data[];
                           ^
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:136:16: note: in the definition of ‘struct nitrokey::proto::EmptyPayload’
         struct EmptyPayload {
                ^~~~~~~~~~~~
In file included from /home/brain/rpmbuild/SPECS/libnitrokey/include/NitrokeyManager.h:6:0,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/NK_C_API.h:6,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/NK_C_API.cc:2:
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:137:27: error: flexible array member ‘nitrokey::proto::EmptyPayload::_data’ in an otherwise empty ‘struct nitrokey::proto::EmptyPayload’
             uint8_t _data[];
                           ^
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:136:16: note: in the definition of ‘struct nitrokey::proto::EmptyPayload’
         struct EmptyPayload {
                ^~~~~~~~~~~~
In file included from /home/brain/rpmbuild/SPECS/libnitrokey/unittest/../include/NitrokeyManager.h:6:0,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/unittest/../NK_C_API.h:6,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/unittest/test_C_API.cpp:8:
/home/brain/rpmbuild/SPECS/libnitrokey/unittest/../include/device_proto.h:137:27: error: flexible array member ‘nitrokey::proto::EmptyPayload::_data’ in an otherwise empty ‘struct nitrokey::proto::EmptyPayload’
             uint8_t _data[];
                           ^
/home/brain/rpmbuild/SPECS/libnitrokey/unittest/../include/device_proto.h:136:16: note: in the definition of ‘struct nitrokey::proto::EmptyPayload’
         struct EmptyPayload {
                ^~~~~~~~~~~~
```

**Comment by szszszsz** [comment]
Hi! @ignatenkobrain 
This is from compilation on G++6, is that right? I have already fixed this, but it awaits to be merged. I will update the issue when this will be done.

**Comment by ignatenkobrain** [comment]
yep, can you give me link to pull request?

**Comment by szszszsz** [comment]
The branch on which is the fix is still WiP, but the commit can be cherry-picked:
https://github.com/Nitrokey/libnitrokey/commit/06fa3e5c8a1756c02b81d28bc18262aa95ba423b

**Comment by szszszsz** [comment]
I have created PR with fix under #50 . Could you check it?
